ChessGame/src/kurs2_task2/Game/Board.java

```diff
@@ -0,0 +1,112 @@
+package kurs2_task2.Game;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.TreeMap;
+
+public class Board {
+	private TreeMap<String, Cell> board = new TreeMap<>();
+	private String[] ABC = { "A", "B", "C", "D", "E", "F", "G", "H"};
+
+
+	public TreeMap<String, Cell> getBoard() {
+		return board;
+	}
+
+	public void setBoard(TreeMap<String, Cell> board) {
+		this.board = board;
+	}
+	public Cell getCell(String cord){
+		return board.get(cord);
+	}
+	public void buildBoard() {
+		String[] tempABC = { "-A", "A", "B", "C", "D", "E", "F", "G", "H", "-H"};

```

Comment:
Why this ABS has same name as constant in the begging of class but has different values? I think it should be named accordingly.

Reply:
Use 'var' instead